frontend/public/components/monitoring/alert-manager-config.tsx

```diff
@@ -189,14 +189,19 @@ const numberOfIncompleteReceivers = (): number => {
   const { route, receivers } = config;
   const { receiver: defaultReceiverName } = route;
 
+  // if no receivers or default receiver, then no longer initial setup, hide info alerts
+  if (!receivers || !defaultReceiverName) {
+    return -1;
+  }
   const defaultReceiver = receivers.filter((receiver) => receiver.name === defaultReceiverName);
   const criticalReceiver = receivers.filter(
     (receiver) => receiver.name === InitialReceivers.Critical,
   );
 
-  const numIncompleteReceivers: number = _.isEmpty(getIntegrationTypes(defaultReceiver[0])) ? 1 : 0;
+  const numIncompleteReceivers: number =
+    !_.isEmpty(defaultReceiver) && _.isEmpty(getIntegrationTypes(defaultReceiver[0])) ? 1 : 0;
```

Comment:
Can this be...?

```suggestion
  const numIncompleteReceivers = getIntegrationTypes(defaultReceiver?.[0]) ? 0 : 1;
```

`: number` is inferred by TypeScript here, so it's not needed.

Prefer just `defaultReceiver &&` to `!_.isEmpty(defaultReceiver)` in most cases (unless we really expect there to sometimes be an empty object).

Reply:
ok, about the typescript.  Yes, `defaultReceiver` can be empty.  That is actually the bug we're fixing.  App. dev deleted all receivers, including the default one, but the defaultReceiver[Name] (defined in `config.routes.receiver` ) was still defined, but no longer any receivers defined in `config.receivers`.

Reply:
fixed, removed `: number`